frontend/public/components/events.jsx

```diff
@@ -136,43 +139,120 @@ export class EventsList extends React.Component {
     super(props);
     this.state = {
       type: 'all',
-      kind: 'all',
       textFilter: '',
+      selected: new Set(['All']),
```

Comment:
nit: why `All` instead of `all`? I have a slight preference for the latter (and it's consistent with `all` for `type` above)

Reply:
No reason.  I will set it to `all`

Reply:
There is a reason, the resource icon expects "All" or else it shows "ALL" which is a change from prior.
![image](https://user-images.githubusercontent.com/18728857/73017850-b6abca80-3ddd-11ea-93b5-1ea59607ceec.png)
I can add all kinds of code to capitalize the first letter, but seems messy.